doc/source/singlevisit.rst

```diff
@@ -0,0 +1,544 @@
+
+These two catalogs provide complimentary views of each field-of-view to try to
+highlight all types of compact sources found in the exposures. 
+
+
+Example Visit
+--------------
+For example, a relatively simple visit of a fairly bright and crowded field with 
+6 F555W exposures (two 15-second and four 30-second exposures) and 
+6 F814W exposures (two 5-second and four 15-second exposures)
+would result in the definition of these output products: 
+
+  * a drizzled image for each separate exposure
+  * a single F555W product
+  * a single F814W product, and
+  * a single **total product**
+  * a point-source catalog for the F555W product
+  * a segmentation-based source catalog for the F555W product
+  * a point-source catalog for the F814W product
+  * a segmentation-based source catalog for the F814W product
+  
+The function ``haputils.poller_utils.interpret_obset_input`` serves as the sole interface 
+for this interpretation. A basic tree gets defined (as a dictionary of dictionaries) 
+by this function where the
+output exposures are identified along with all the names of the input exposures.
+This tree then serves as the basis for organizing the rest of the SVM processing.
+
+In addition to defining what output products need to be generated, all the SVM
+products names are defined using the :ref:`svm_naming_convention`.  This insures
+that all the output products have filenames which are not only unique but also 
+understandable (if a bit long) that are easily grouped on disk.  
+
+
+Aligning the Input Data
+=======================
+All input exposures should have already been aligned either individually or by 
+association table as close to GAIA as possible during standard pipeline calibration
+processing.  However, each exposure or association (of exposures) can be aligned
+to slightly different fits or catalogs due to differences in the source objects 
+which can be identified in each separate exposure.  The primary goal of SVM 
+processing is to refine this alignment so that all exposures in the visit for 
+the same detector (those exposures which contribute to each **total product**)
+share the same WCS (pixels on the sky).  
+
+Alignment of all the exposures for a **total product** uses the same alignment
+code as the standard calibration pipeline.  The basic steps it follows is:
+
+  * generate a source catalog for each exposure (using :ref:`amutils_api`)
+  * obtain the WCS from each exposure
+  * perform a relative fit between the exposures using ``tweakwcs``
+  * obtain an astrometric catalog for the field-of-view 
+  * perform a final fit of all the exposures at once to the astrometric catalog
+  * update each WCS with the final corrected WCS generated by ``tweakwcs``
+  
+The limits for performing the relative alignment and absolute fit to the astrometric
+catalog (defaults to **GAIADR2**) are lower under the expectation that large 
+offsets (> 0.5 arcseconds) have already been removed in the pipeline.  This makes
```

Comment:
have already been removed in the pipeline.

Change to "have already been removed in the pipeline processing."